### PR TITLE
Update security non-events more.

### PR DESCRIPTION
**Pull request by kentonv**
Added some non-non-events, for good measure.

### CODE REVIEW

File: docs/using/security-non-events.md [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
-* Of course, Sandstorm itself could be buggy, but Sandstorm is written by security nerds -- the kind of people who do ask themselves "What is every possible path this code could take?" Moreover, we plan to commission an outside security review before version 1.0. Hopefully, the chance of bugs in Sandstorm is much less than the chance of bugs in an average app.
-* The list below is not even remotely complete. Rather, it is a sample of the few bugs we've actually researched -- from the few apps that are big enough to have attracted security review and publicly disclosed vulnerabilities.
+* Sandstorm is NOT a "web application firewall". WAFs monitor for and block known attacks and suspicious behavior. Sandstorm uses fine-grained containerization and access control to mitigate vulnerabilities across the board, including vulnerabilities that haven't been discovered yet. See [how Sandstorm works](https://sandstorm.io/how-it-works) and [Sandstorm's security practices](security-practices) for details.
```
Comment (paulproteus) [inline docs/using/security-non-events.md]: Semi-troll: Sandstorm is a web app firewall! It's just one that operates at completely insane scale!

Proposed:

s/Sandstorm is NOT a "web application firewall"./Sandstorm does NOT operate like a traditional "web application firewall"./

Not a show-stopper, but fwiw.

Comment (paulproteus) [inline docs/using/security-non-events.md]: One way to misunderstand Sandstorm while believing only true things is that "Sandstorm is a web application firewall that adds HTTP firewall rules that amount to access control checks." I agree that's not really the point of a web app firewall, though.

Comment (kentonv) [inline docs/using/security-non-events.md]: OK, I tweaked the wording.

File: docs/using/security-non-events.md [hunk 2]
```diff
@@ -69,3 +99,7 @@ Some bugs blocked include:
...
 * [CVE-2016-0728](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-0728): Use-after-free caused by crafted `keyctl()` calls could lead to privilege escalation. Sandstorm disables `keyctl()` inside the sandbox using seccomp.
 
+Bugs *not* mitigated:
+
+* [CVE-2014-9090](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-9090), [9322](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-9322): AKA "BadIRET". According to Andy Lutomirski, who discovered and fixed the bug (and who wrote Sandstorm's seccomp filter): "Very hard to exploit from inside Sandstorm, but it just might have been possible using a bizarre vector. Certainly the standard exploit would not work."
```
Comment (paulproteus) [inline docs/using/security-non-events.md]: Mark Seaborn would say we should mention rowhammer, I guess? Feel free to leave it out too I guess.

Comment (paulproteus) [inline docs/using/security-non-events.md]: ...though I guess that's for a section called "Your computer" not "Linux kernel".

## Your computer and physical reality

We can't protect against people with:

- A screwdriver and physical access, willing to remove your hard drive

- Rowhammer

- People looking over your shoulder with a video camera and watching you type your password

**Comment by paulproteus** [comment]
Thanks. I'm +1 on this, going to merge & push now.

**Comment by paulproteus** [comment]
https://docs.sandstorm.io/en/latest/using/security-non-events/ is live, yay.

File: docs/using/security-non-events.md (hunk 2 above)
Comment (kentonv) [inline docs/using/security-non-events.md]: I actually considered adding rowhammer but ultimately decided that it didn't belong.

**Pull request merged**